packages/react/runtime/src/lynx/performance.ts

```diff
@@ -22,8 +22,19 @@ enum PerformanceTimingKeys {
   patch_changes_end,
   hydrate_parse_snapshot_start,
   hydrate_parse_snapshot_end,
+  mtsRenderStart,
+  mtsRenderEnd,
 }
 
+const PerformanceTimingFlags = {
+  reactLynxHydrate: 'react_lynx_hydrate',
+};
```

Comment:
To make it more strict in type, use `as const`

```suggestion
} as const;
```